### DISCUSSION TITLE
请问最轻模式本地部署方案要怎么部署

### DISCUSSION THREAD

**Discussion by AkimioJR** [chatchat-space/Langchain-Chatchat#1966] [General]
请问最轻模式本地部署方案要怎么部署，仓库中好像已经没有`requirements_lite.txt`这个文件了，`startup.py`也没有`--lite`的启动方式。
我想在NAS上搭建一个WebUI，后端大模型用ChatGPT，请问我应该怎么操作
![image](https://github.com/chatchat-space/Langchain-Chatchat/assets/77880495/74849b11-7b4e-4379-aab0-66accfc5b1f7)

**Comment by zRzRzRzRzRzRzR** [comment]
更新到0.2.7试试，那边有轻量化的
